Checked type to find type (#678)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

> * **validate** and **secrets** commands are now using a new find type function without regex.

Reviewer: ```suggestion
* Updated the **validate** and the **secrets** commands to be less dependent on regex.
```
